frontend/src/presentation/utils/constants.ts

```diff
@@ -41,10 +43,10 @@ export const QUESTIONS_FILTER_TEXT = {
 }
 
 export const DIFFICULTY_TEXT = {
```

Comment:
I feel that this constant is redundant if we already have QuestionDifficulty. We should keep either one

Reply:
Nah, we need this because this text includes "All" which doesn't exist for your difficulty options.